wpiunits/generate_units.py

```diff
@@ -9,25 +9,17 @@
 # Generated files will be located in wpiunits/src/generated/main/
 
 import inspect
-import os
 import re
 from jinja2 import Environment, FileSystemLoader
+from pathlib import Path
+import argparse
+import sys
```

Comment:
These should be above the inspect import.

Reply:
Why?

Reply:
The imports should be properly sorted. Standard imports go first, then thirdparty, alphabetical.